src/systems/filecoin_vm/runtime/runtime.go

```diff
@@ -366,7 +366,7 @@ func (rtOuter *VMContext) _sendInternal(input InvocInput, ignoreErrors bool) msg
 	rtOuter._refundGasRemaining(toActorMethodGasBound)
 	rtOuter._deductGasRemaining(gasUsed)
 
-	if !ignoreErrors && invocOutput.ExitCode().IsError() {
+	if !catchErrors && invocOutput.ExitCode().IsError() {
```

Comment:
To make sure I understand this, as the line suggests, if it is not catching error and there is an error, this function will throw. Correct.